src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionContainerBase.cs

```diff
@@ -620,22 +630,25 @@ private TimerAwaitable Init()
 
             private async Task PingAsync(TimerAwaitable timer)
             {
-                while (await timer)
+                using (timer)
```

Comment:
`using` timer will lead to dispose error, that when finally remove endpoint, we'll dispose the container and call `Container.Dispose()`, which will call timer.Dispose() and will throw stop a disposed item.